examples/manifests/hw-event-proxy.yaml

```diff
@@ -50,12 +55,14 @@ spec:
                 secretKeyRef:
                   name: redfish-basic-auth
                   key: hostaddr
+            - name: LOG_LEVEL
+              value: "debug"
         - name: cloud-native-event-sidecar
           image: quay.io/redhat_emp1/cloud-event-proxy:latest
           args:
             - "--metrics-addr=127.0.0.1:9091"
             - "--store-path=/store"
-            - "--transport-host=amqp://amq-interconnect.$(AMQP_NAMESPACE).svc.cluster.local"
+            - "--transport-host=amqp://router.$(AMQP_NAMESPACE).svc.cluster.local"
             - "--api-port=8080"
```

Comment:
change this to 8089 or any other value. 8080 is very generic and might cause conflict

Reply:
done